pep-0458.txt

```diff
@@ -457,6 +457,17 @@ pip for the first time.
 
 __ https://docs.google.com/spreadsheets/d/11_XkeHrf4GdhMYVqpYWsug6JNz5ZK6HvvmDZX0__K2I/edit?usp=sharing
 
+This number of bins SHOULD double when the metadata overhead for returning
+users exceeds 50% (relative to the average size of downloaded packages).
+Presently, this SHOULD happen when the number of targets (simple indices
+and packages) increase at least 4x from over 2M to nearly 9M, at which
+point the metadata overhead for returning and new users would be around
+49-54% and 185% respectively, assuming that the number of bins stay fixed.
```

Comment:
Are these the worst case numbers? I think we should make that clear because it might affect when doubling should occur in practice

Reply:
Sorry, could you clarify what "worst case numbers" means?

Reply:
They assume that all filenames, etc are the maximum length

Reply:
I agree with Marina. But IMO pointing to the numbers, as done in [L458](https://github.com/secure-systems-lab/peps/pull/29/files#diff-7ffbd592926c74789442ddd1465bb9e0R458), should be enough.

Reply:
OK, I'll clarify